drf_spectacular/openapi.py

```diff
@@ -723,7 +723,10 @@ def _map_serializer_field(self, field, direction, bypass_extensions=False):
             return append_meta(build_array_type(build_choice_field(field)), meta)
 
         if isinstance(field, serializers.ChoiceField):
-            return append_meta(build_choice_field(field), meta)
+            schema = build_choice_field(field)
+            if 'description' in meta:
+                meta['description'] = meta['description'] + '\n\n' + schema.pop('description')
```

Comment:
This generates e `KeyError` if `schema` does not have `description`. You need to use `schema.pop('descriotion, None)`

Reply:
@tfranzel  this is breaking our current build, because we set `"ENUM_GENERATE_CHOICE_DESCRIPTION": False`

Reply:
Damn, can't seem do a release without a `.1` apparently. 

> You need to use schema.pop('description', None)

would lead to a None str concat `TypeError`, but I get the point.

Reply:
Will wait for the release until tonight to see whether something else will come up.